code/modules/events/merchant/merchant_beacon.dm

```diff
@@ -0,0 +1,47 @@
+/obj/machinery/satellite/merchant
+	name = "trading satellite"
+	desc = "A satellite broadcasting a trading signal, to indicate the station is open for business."
+	icon = 'icons/obj/machines/satellite.dmi'
+	icon_state = "sat_merchant_inactive"
+	base_icon_state = "sat_merchant"
+
+	mode = "BIGMONEYV.1"
+	active = FALSE
+
+	///minimum countdown ticks
+	var/countdown_min = 1 MINUTES
+	///maximum countdown ticks
+	var/countdown_max = 4 MINUTES
+	///the current countdown till the merchant event is triggered
+	var/countdown
+	///how much the countdown goes down per tick
```

Comment:
per second, not per tick. what a "tick" is is subject to change (which is why you need to use delta time)